server/src/main/java/org/elasticsearch/index/mapper/MapperFeatures.java

```diff
@@ -71,7 +73,8 @@ public Set<NodeFeature> getTestFeatures() {
             IgnoredSourceFieldMapper.IGNORED_SOURCE_AS_TOP_LEVEL_METADATA_ARRAY_FIELD,
             IgnoredSourceFieldMapper.ALWAYS_STORE_OBJECT_ARRAYS_IN_NESTED_OBJECTS,
             MapperService.LOGSDB_DEFAULT_IGNORE_DYNAMIC_BEYOND_LIMIT,
-            CONSTANT_KEYWORD_SYNTHETIC_SOURCE_WRITE_FIX
+            CONSTANT_KEYWORD_SYNTHETIC_SOURCE_WRITE_FIX,
+            ERROR_CODE_CHANGED
```

Comment:
I wonder if we should include some more details about the change in the constant name, like `META_FETCH_FIELDS_ERROR_CODE_CHANGED`

Reply:
Thank you, @javanna; the name you proposed looks better.